identity-apps-core/apps/recovery-portal/src/main/webapp/username-recovery.jsp

```diff
@@ -190,7 +190,15 @@
                 <h2><%=i18n(recoveryResourceBundle, customText, "username.recovery.heading")%></h2>
                 <% if (error) { %>
                     <div class="ui visible negative message" id="server-error-msg">
-                        <%= IdentityManagementEndpointUtil.i18nBase64(recoveryResourceBundle, errorMsg) %>
+                        <% if (IdentityRecoveryConstants.ErrorMessages.ERROR_CODE_MULTIPLE_MATCHING_USERS.getCode().equals(errorCode)) {
+                        %>
+                            <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "multiple.users.found")%>
+                        <% } else if (IdentityRecoveryConstants.ErrorMessages.ERROR_CODE_NO_USER_FOUND.getCode().equals(errorCode)) {
+                        %>
+                            <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "no.user.found")%>
```

Comment:
Let's use the i18n function from localize.jsp instead. Please check and update other places too.

Ref: https://github.com/KaveeshaPiumini/identity-apps/blob/i18n-backend-errors/identity-apps-core/apps/recovery-portal/src/main/webapp/username-recovery.jsp#L190